<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sales_report</class>
 <widget class="QWidget" name="sales_report">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Month</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Print Report</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Export as csv</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>product</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="5">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Sales Report&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="5">
      <widget class="QTableWidget" name="tableWidget">
       <column>
        <property name="text">
         <string>Date</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Product</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Qty</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Total</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDateEdit" name="dateEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
